### PR TITLE
login api and save refresh token in db

### PR DESCRIPTION
**Pull request by Bilal-CodeMaster**

### CODE REVIEW

**Comment by Bilal-CodeMaster** [comment]
adill login complete please review it

File: controller/auth.Controller.js [hunk 1]
```diff
@@ -3,12 +3,29 @@ import { ApiError } from "../utils/ApiError.js";
 import { ApiResponse } from "../utils/ApiResponse.js";
 import { tryCatchHandler } from "../utils/tryCatchHandler.js";
 
-class AuthController {
-  static signIn = () => {};
+const generateAccessAndRefreshToken = async (userId) => {
```
Comment (adil-techcode) [inline controller/auth.Controller.js:6]: this is good , but we can use also another class or method in utils for generate tokens

**Review by adil-techcode — COMMENTED** [review]
Good Work Bilal

File: controller/auth.Controller.js (hunk 1 above)
Comment (Bilal-CodeMaster) [reply controller/auth.Controller.js:6]: ok

**Pull request merged**